Tasks/NuGetInstaller/task.loc.json

```diff
@@ -9,7 +9,7 @@
   "version": {
     "Major": 0,
     "Minor": 2,
-    "Patch": 20
+    "Patch": 23
```

Comment:
Can we go to 21?  I left 20/21 for R105 ... silly to only leave one extra but I didn't like how much our versions were bouncing around but couldn't justify a minor roll.  There is a pending PR to master with .23 so if not .21 it would be great if you could .24.  I hate the tracking this involves.